### DISCUSSION TITLE
Installation Dashboard error

### DISCUSSION THREAD

**Discussion by BenMauriceKeil** [nfrastack/container-freescout#193] [Q&A]
We have entered all the Docker variables that need to be entered, but we still cannot get to the login screen, we only get an error message.
<img width="1546" alt="Screenshot 2024-10-23 193126" src="https://github.com/user-attachments/assets/b2208b56-c420-4c0d-baa5-e246cb1d8a11">

The following variables were entered in Docker
ADMIN_EMAIL
ADMIN_FIRST_NAME
ADMIN_LAST_NAME
ADMIN_PASS
APPLICATION_NAME
DB_TYPE
DB_HOST
DB_PORT
DB_NAME
DB_USER
DB_PASS
ENABLE_AUTO_UPDATE
SETUP_TYPE
SITE_URL

**Comment by GCGreg2** [comment]
Hi Ben, did you resolve this in the end?

I'm getting the same, looking at the logs it's erroring on trying to access the database with the username freescout@172.0..... for me...

cheers

**Comment by GCGreg2** [comment]
OK so this came down to one or an unholy combination of all 3 of these factors:

1) didn't setup groups:

https://docs.docker.com/engine/install/linux-postinstall/

2) used  the same password for ROOT and DB

3) had a special character in the above
